### DISCUSSION TITLE
How to export and import using ENV variables KC_REALM, KC_DIR, KC_USERS

### DISCUSSION THREAD

**Discussion by HaritSharma16** [keycloak/keycloak#20310] [Archive]
Hi All,

I wanted to export a realm along with the user information, for which I am using env variables  KC_REALM, KC_DIR, KC_USERS. I am passing these variable as part of my helm upgrade command, and after upgrade I can see these variables present as part of my keycloak pod, but I cant see the already present realm exported to a json, that is json is not at all created.

How can I export a realm along with user info and then import the same json for having same structure of realm along with users.
I will be awaiting responses!!

Thanks!!

**Comment by vmuzikar** [comment]
@HaritSharma16 Hello, I believe you need to use the `export` command. Please see the relevant [guide](https://www.keycloak.org/server/importExport).
